mne_bids/mne_bids.py

```diff
@@ -538,7 +578,14 @@ def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
         A path to the configuration file to use if the data is from a BTi
         system.
     overwrite : bool
-        If the file already exists, whether to overwrite it.
+        Defaults to False.
+        Whether to overwrite existing files or data in files.
+        If overwrite == True, any existing files will be overwritten with the
```

Comment:
I'd use plain English here and say `If overwrite is True`

also it's not "any" existing file but rather the BIDS files matching the parameters provided.

Reply:
ah yeah true. Thanks